.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,66 +24,62 @@
 
 <!-- Please add direct links where your modifications can be seen in the documentation -->
 
-* https://deploy-preview-{your pr number}--boosted.netlify.app/
+* https://deploy-preview-{your_pr_number}--boosted.netlify.app/
 
 ### Checklist
 
 <!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 
+#### Contribution
+
 - [ ] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
+
+#### Design
+
+- [ ] My change respects the design guidelines
+- [ ] My change is compatible with responsive display
+
+#### Development
+
 - [ ] My code follows the code style of the project _(using `npm run lint`)_
-- [ ] My change introduces changes to the documentation
-- [ ] My change introduces changes to the migration guide
-- [ ] I have updated the documentation accordingly
-- [ ] I have added tests to cover my changes
+- [ ] I have added JavaScript unit tests to cover my changes
 - [ ] All new and existing tests passed
 
-<!------------------------->
-<!-- /!\ Core Team Only -->
-<!------------------------->
+#### Documentation
 
-<!-- Uncomment the following for a feature DoD -->
+- [ ] My change introduces changes to the documentation or I have updated the documentation accordingly
```

Comment:
and rather than or

Reply:
Changed it to "and/or" in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1706/commits/2eb12c29a488d7646189ee4358efe95b88e95779